tests/test_utils.py

```diff
@@ -259,5 +261,26 @@ def test_get_journal_and_section_invalid():
     publication = ""
     journal_title, section = get_journal_and_section(publication)
 
-    assert journal_title == ''
-    assert section == ''
+    assert journal_title == ""
+    assert section == ""
+
+
```

Comment:
I like the idea, but I didn't change it because I think it makes the code look unnecessarily complicated. Are there some performance improvements here? I can change it if you really want it :)